docs/implementations/watt-time.md

```diff
@@ -1,15 +1,82 @@
 # WattTime Grid Emissions Model
 
-# Introduction
+## Introduction
 
 WattTime technology—based on real-time grid data, cutting-edge algorithms, and machine learning—provides first-of-its-kind insight into your local electricity grid’s marginal emissions rate. [Read More...](https://www.watttime.org/api-documentation/#introduction)
 
 
-# Scope 
+## Scope 
 
 WattTime Model provides a way to calculate emissions for a given time in a specific location. 
 
 The model is based on the WattTime API. The model uses the following inputs:
-* location: Location of the software system ({latitude:0.0, longitude:0.0})
+* latitude: Location of the software system (latitude in decimal degrees).
+* longitude: Location of the software system (longitude in decimal degrees).
 * timestamp: Timestamp of the recorded event (2021-01-01T00:00:00Z) RFC3339
 * duration: Duration of the recorded event in seconds (3600)
+
+
+## Implementation
+
+Limitations: 
+* Set of observations are to be within 32 days of each other. 
+* Emissions are aggregated for every 5 minutes regardless of the granularity of the observations.
+
+### Authentication
+
+
+WattTime API requires activation of subscription before usage. Please refer to WattTime website for more information.
+
+**Required Parameters:**
+
+* username: Username for the WattTime API
```

Comment:
I can see from your code that if the username is an ENV var then it pulls the value from the ENV. This is what I asked you to document!!?

Reply:
This is exactly what I asked you to document in my last review, why are you asking me to review again??? I don't have time for this.